projects/vdk-plugins/vdk-control-api-auth/.plugin-ci.yml

```diff
@@ -3,29 +3,60 @@
 
 image: "python:3.7"
 
-# TODO: Uncomment when committing new changes
-#.build-vdk-control-api-auth:
-#  variables:
-#    PLUGIN_NAME: vdk-control-api-auth
-#  extends: .build-plugin
-#
-#build-py37-vdk-control-api-auth:
-#  extends: .build-vdk-control-api-auth
-#  image: "python:3.7"
-#
-#
-#build-py38-vdk-control-api-auth:
-#  extends: .build-vdk-control-api-auth
-#  image: "python:3.8"
-#
-#
-#build-py39-vdk-control-api-auth:
-#  extends: .build-vdk-control-api-auth
-#  image: "python:3.9"
-#
-#build-py310-vdk-control-api-auth:
-#  extends: .build-vdk-control-api-auth
-#  image: "python:3.10"
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
+
+.build-vdk-control-api-auth:
```

Comment:
Why not reuse .build-plugin from plugin-common ?

Reply:
@tozka Updated the main plugin CI/CD script to ignore the entry-point of the plugin, similar to how `quickstart-vdk` and `vdk-test-utils` are handled. We may need to think about a different way to handle such plugins, though, as this does not seem manageable in the long run.

Reply:
Yes, I agree, it's not ideal. 2 options come to mind: 

A) We remove the check. If we think what the check is doing (testing the "wiring") is covered by other tests . 
B) Make the check option but reverse - that is expose variable (CHECK_VDK_PLUGIN_ENTRYPOINT=false/true) and those 3 plugins/libs would set it to false.